### PR TITLE
fix: format compiler errors (#28)

### PR DESCRIPTION
**Pull request by tungpin**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   "bugs": "https://github.com/smooth-code/error-overlay-webpack-plugin/issues",
   "homepage": "https://github.com/smooth-code/error-overlay-webpack-plugin",
   "repository": "https://github.com/smooth-code/error-overlay-webpack-plugin.git",
-  "version": "0.1.6",
+  "version": "0.1.7",
```
Comment (neoziro) [inline package.json]: Please do not increment the version, I will do it in the release process.

Comment (tungpin) [reply package.json]: Ok... in addition I found some errors on CI，it looks like caused by you not install dependencies of example

Comment (neoziro) [reply package.json]: Hmm.. OK could you add them?

Comment (tungpin) [reply package.json]: of course

**Pull request merged**